compose.object-storage.yml

```diff
@@ -0,0 +1,64 @@
+services:
+  ghost:
+    extends:
+      file: compose.yml
+      service: ghost
+    profiles: [object-storage]
+    environment:
+      - storage__media__adapter=S3Storage
+      - storage__media__staticFileURLPrefix=content/media
+      - storage__files__adapter=S3Storage
+      - storage__files__staticFileURLPrefix=content/files
+      - storage__S3Storage__bucket=ghost-dev
+      - storage__S3Storage__region=us-east-1
+      - storage__S3Storage__prefix=ab/ab1234567890abcdef1234567890abcd
```

Comment:
Should this be `storage__S3Storage__tenantPrefix` to match the [S3StorageOptions](https://github.com/TryGhost/Ghost/pull/25435/files#diff-a0465012b65b5d951b8b4dd286e53d0f1685ea7a8a7cf4eb56109877f221f495R49)?